src/Dfe.PlanTech.Web/Controllers/RecommendationsController.cs

```diff
@@ -29,6 +30,31 @@ public class RecommendationsController(ILogger<RecommendationsController> logger
           cancellationToken);
     }
 
+    [HttpGet("{sectionSlug}/recommendation/{recommendationSlug}/preview/{maturity?}", Name = "GetRecommendationPreview")]
```

Comment:
Not a big issue, but it feels a bit odd to have `section/recommendation/wireless-network-low/preview/high` as a url, where the maturities conflict, and you've already provided it with the slug
would it make more sense for this to be 

`[HttpGet("{sectionSlug}/recommendation/preview/{maturity?}", Name = "GetRecommendationPreview")]` instead and just omit the recommendation slug?

Reply:
I considered something like this, especially because technically the `recommendationSlug` isn't even used in either route.

However, I believe (although haven't tested), that `{sectionSlug}/recommendation/preview/{maturity?}` will conflict with the main route. If `maturity` is null then there's no way to differentia between the routes, as `preview` could be a value for `recommendationSlug`.

Gonna test this out though.

Reply:
No conflicts per se, but it always tries to use the original route and then complains about lack of slug.

Reply:
Ah I thought it would look at the /preview one first and then fall back to the templated one, 
It tries to use the /recommendationSlug route?

Reply:
I've made the change you suggested FWIW

Reply:
I see your point with it, didn't mean to push into a choice you didn't like, not set on it, I just think this is a more natural route as it doesn't repeat the maturity and allow different ones, and we could reroute to the page as the slug isn't used but I think the homepage as you've done is fine - defo like the maturity bit you added 👍